### ISSUE TITLE
Can't build on RN 0.32 (LoadImageWithTag removed)

### ISSUE THREAD

**Issue by samuelkraft** [naoufal/react-native-activity-view#57]
LoadImageWithTag was deprecated a while ago and [removed with rn 0.32](https://github.com/facebook/react-native/commit/7e5de707be0d0bdfe90f908b1fc7d6de08050613). It is now not possible to build with Xcode anymore.

**Comment by ammichael** [comment]
No workaround yet?
I need to send an update to the store, and this is the only lib I can't get to work

**Comment by ismdcf** [comment]
thnks @ammichael  is this updated with the latest release or should I have to use your commit to install it

**Comment by ammichael** [comment]
@ismdcf actually the PR is not mine. I just changed the same line, in the same file, and worked like a charm.

About your error, seems like the problem is something in your code (but hey, I may be wrong)

**Comment by digitalcatnip** [comment]
@ismdcf Not sure if you solved the issue, but it looks like loadImageWithURLRequest requires an NSURLRequest, and the existing code passes an NSString, causing the error.

This code fixes the issue:

`NSURL *url = [NSURL URLWithString:imageUrl];`
`NSURLRequest *request = [[NSURLRequest alloc] initWithURL:url];`
`[loader loadImageWithURLRequest:request callback:^(NSError *error, id imageOrData) {`

**Comment by ismdcf** [comment]
yep it did not work .Tk @digitalcatnip I'll test and let u know .
